packages/frontend/core/src/pages/workspace/detail-page/detail-page.tsx

```diff
@@ -83,6 +83,7 @@ const DetailPageImpl = memo(function DetailPageImpl() {
   const globalContext = useService(GlobalContextService).globalContext;
   const docCollection = workspace.docCollection;
   const mode = useLiveData(doc.mode$);
+  const isSideBarOpen = useLiveData(workbench.sidebarOpen$);
```

Comment:
Do we need to check the tab opened by the sidebar here? In addition, in splitview mode, the sidebar may be displaying the content of other views. Do we need more specific controls?

Reply:
Yes, the sidebar is always hidden when the sidebar is opened.